book_source/02_demos_tutorials_workflows/04_more_web_interface/02_hidden_analyses.Rmd

```diff
@@ -467,8 +467,37 @@ An example of multi-settings pecan xml file also may look like below:
 </pecan.multi>
 ```
 
+### Running SDA on remote
+In general, the idea is that sending, running and monitoring a SDA job should be all done using just two functions (`SDA_remote_launcher` and `Remote_Sync_launcher`). `SDA_remote_launcher` checks for the XML settings defining the run and sets up the SDA on the remote machine and sends a qusb command for running the job. `Remote_Sync_launcher` on the other hand sits on the local machine and monitors the progress of the job and brings back the outputs as long as the job is running. 
+
+`SDA_remote_launcher` sets up the job by copying a template SDA workflow R script as well as a bash file template ready for submission to the remote machine. This functions checks the paths to the all inputs including met, soil, `site_pft` and etc testing whether they exists on the remote machine or not; If they don't it copies them over and replace the settings accordingly. After submitting the bash script it returns the PID of the job writing the log file, allowing the `Remote_Sync_launcher` to monitor the progress of the run and check to see if the job is still alive and if the `sda.output.rdata` is being updated since the last check or not.
```

Comment:
```suggestion
`SDA_remote_launcher` sets up the job by copying a template SDA workflow R script and a bash file template that are ready for submission to the remote machine. This function checks the paths to all inputs including met, soil, `site_pft` and etc., testing whether they exists on the remote machine or not. If they do not exist, the function copies the missing paths over and replaces the settings accordingly. After submitting the bash script, the function returns the PID of the job writing the log file, allowing the `Remote_Sync_launcher` to monitor the progress of the run, checks to see if the job is still alive, and determines if `sda.output.rdata` has been updated since the last check or not.
```